core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -777,7 +777,7 @@ private void sendToAll(ReplicableCommand rpcCommand, DeliverOrder deliverOrder)
          AnycastAddress anycastAddress = new AnycastAddress();
          dispatcher.sendMessage(anycastAddress, buffer, options);
       } else {
-         dispatcher.castMessage(null, buffer, options);
+         dispatcher.castMessage(null, buffer, options.anycasting(false));
```

Comment:
Looks superfluous. the default is already `false`. It actually made me think maybe we're setting it to `true` earlier, and that we shouldn't do that :)